src/commands/build/upload.rs

```diff
@@ -325,7 +325,13 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
             }
         );
         for (path, reason) in errored_paths_and_reasons {
-            warn!("  - {} ({})", path.display(), reason);
+            warn!("  - {}", path.display());
+            if let Some(source) = reason.source() {
+                warn!("    Error: {}", reason);
+                warn!("    Cause: {}", source);
+            } else {
+                warn!("    Error: {}", reason);
+            }
```

Comment:
I think this is what you want 🙂

```suggestion
            warn!("    Error: {reason:#}");
```

The important part is the `:#`, which tells `anyhow` to use its [alternate `Display` format](https://docs.rs/anyhow/latest/anyhow/struct.Error.html#display-representations), which includes the cause chain.

Reply:
Yes and no. That’s a separate bug we noticed where the cli tries to automatically detect the provider but it sends up `github.com` instead of `github`. The server only accepts `github`. I have a separate PR for that.